### ISSUE TITLE
Content label contrast issue

### ISSUE THREAD

**Issue by kevindherman** [proudcity/wp-proudcity#2573]
## Source

* [Support ticket](https://proudcity.zendesk.com/agent/tickets/6989) 

## Describe the bug
Document type label doesn't have enough contrast being gray and white. This label is used for all content type labels, not just documents. It either needs to be a darker gray or be black. 

## To reproduce
https://www.santa-ana.org/documents
![Screen Shot 2024-06-26 at 3 20 06 PM](https://github.com/proudcity/wp-proudcity/assets/6032426/620f3d89-25ba-47ac-a74b-07f7dc3f789c)

## Additional context
@lukefretwell do you think the label should be a darker gray or black as a platform standard?

## QA

Notes:

* 

Links:

*

**Comment by lukefretwell** [comment]
@kevindherman I'd suggest a lighter gray and use the primary link color.

This may already be documented so might want to check.

**Comment by kevindherman** [comment]
Just checked, don't see an open issue for this one.

We recently improved jobs labels to match all other content labels with this closed issue: https://github.com/proudcity/wp-proudcity/issues/2542
